pkg/client/client.go

```diff
@@ -310,6 +310,11 @@ func FormatStatusResponse(w io.Writer, sr *models.StatusResponse, sd StatusDetai
 		}
 		fmt.Fprintf(w, "\n")
 	}
+
+	if sr.CniChaining != nil {
```

Comment:
Feel free to let me know where cni chaining output should be in `cilium status`.

Reply:
Location seems fine to me. I don't think we have explicit rules on the order of things :shrug: